src/org/zaproxy/zap/extension/httppanel/HttpPanel.java

```diff
@@ -210,7 +210,6 @@ public void actionPerformed(ActionEvent e) {
                     if (focusOwner instanceof JTextComponent) {
                         FindDialog findDialog = new FindDialog(SwingUtilities.getWindowAncestor(focusOwner), false);
```

Comment:
Instead of creating a new `FindDialog` it should be requested one (e.g. `FindDialog.getDialog(Window)`, that returns the existing one or creates one if none exists), otherwise the following statement has no effect if the dialogue is already open.